content/jacoco/index.html

```diff
@@ -428,4 +428,4 @@ <h2>Contact/Feedback</h2>
 </ul>
 
 </body>
-</html>
\ No newline at end of file
+</html>

```

Comment:
I made these changes direct on GitHub UI. The editor UI that added this new line 🤷‍♂️

Reply:
No prob, I fixed it.